features/FxA-57-verify-reminder/README.md

```diff
@@ -0,0 +1,40 @@
+
+Verification Reminder Email Specification
+======================
+
+https://mozilla.aha.io/features/FXA-57
+
+As a new Firefox Accounts user,
+ I have signed up for an FxA account, but did not verify my email. 
+ 
+ I should get 2 email reminders that will ask me to verify my email.
+ 
+ The first email arrives after one day since sign up.
+ 
+ The second email arrives after three days since sign up.
+ 
+## Basic Email Template
+
+> This is just a visual guide, please see detailed email content below
+
+![](basic-template.png)
+
+
+## Email 1
+
+### Header: Hello again.
+
+### Content:
+
+A few days ago you created a Firefox Account, but never verified it. 
+Please activate your account by confirming this email address.
+
+
+## Email 2
+
+### Header: Still there?
+
+### Content:
+
+A few weeks ago you created a Firefox Account, but never verified it.
```

Comment:
"A few weeks" is not consistent with "the second email arrives after three days since sign up".